internal/magic/image.go

```diff
@@ -104,3 +104,8 @@ func Jxl(raw []byte, _ uint32) bool {
 	return bytes.HasPrefix(raw, []byte{0xFF, 0x0A}) ||
 		bytes.HasPrefix(raw, []byte("\x00\x00\x00\x0cJXL\x20\x0d\x0a\x87\x0a"))
 }
+
+// Jxr matches JPEG XR image file
+func Jxr(raw []byte, _ uint32) bool {
```

Comment:
Please use the `prefix` function for this because it's shorter. See, for example, the [function for PNG files](https://github.com/gabriel-vasile/mimetype/blob/e7130b0cc5867df18adc9baa5d4b1a5419b039bf/internal/magic/image.go#L8).